pkg/trigger/aws/handler.go

```diff
@@ -29,17 +30,25 @@ func (h *HookHandler) Loop(nodeName string) {
 			continue
 		}
 		glog.Infof("Status of instance '%v' is '%v', autoscaling group is '%v'", h.AutoScaling.Options.InstanceID, *status, *autoScalingGroupName)
-		if !h.AutoScaling.IsTerminating(status) {
+		if !h.AutoScaling.IsTerminating(status) && !h.AutoScaling.IsTerminatingWait(status) {
```

Comment:
I love the fact that you've used both to make sure it triggers if the docs and impl from AWS diverge.